docs/source/user_guide/benchmarks/surfaces.rst

```diff
@@ -101,3 +101,47 @@ Reference data:
 
 * Same as input data
 * PBE-D3(BJ), MPRelaxSet settings
+
+Elemental Slab Oxygen Adsorption
+================================
+
+Summary
+-------
+
+Performance in predicting adsorption energies for oxygen on elemental slabs.
+
+Metrics
+-------
+
+Adsorption energy error
+
+For each slab, two single points are performed.
+The first is for the isolated slab and the second is the slab with
+an oxygen placed on-top of the site furthest along the slab's normal
+direction. The distance between the oxygen and the site is that which
+minimizes the energy according to MACE-MatPES-r2SCAN.
+
+Computational cost
+------------------
+
+Very low: tests are likely to take less than a minute to run on CPU.
+
+Data availability
+-----------------
+
+Input data:
+
+* Elemental slabs were obtained using the Materials Project API. The lowest-surface-energy slab
+  of the most stable experimentally observed bulk crystal is chosen. The relaxed slabs were
+  submitted to the Materials Project by:
+
+  * R. Tran, Z. Xu, B. Radhakrishnan, D. Winston, W. Sun, K. A. Persson, S. P. Ong, "Surface Energies of Elemental Crystals," Scientific Data, 2016, 3:160080. https://doi.org/10.1038/sdata.2016.80
+
+Reference data:
+
+* Single points are performed with the MatPESStatic set, with a cutoff energy of 520 eV.
+
+  * A. D. Kaplan, R. Liu, J. Qi, T. W. Ko, B. Deng, J. Riebesell, G. Ceder, K. A. Persson, S. P. Ong, "A Foundational Potential Energy Surface Dataset for Materials," arXiv preprint arXiv:2503.04070, 2025. https://doi.org/10.48550/arXiv.2503.04070
+  * S. P. Ong, W. D. Richards, A. Jain, G. Hautier, M. Kocher, S. Cholia, D. Gunter, V. Chevrier, K. A. Persson, G. Ceder, "Python Materials Genomics (pymatgen): A Robust, Open-Source Python Library for Materials Analysis," Comput. Mater. Sci., 2013, 68, 314–319. https://doi.org/10.1016/j.commatsci.2012.10.028
+
+* Tran et al. relaxed the slabs using spin-polarized PBE calculations performed in VASP, with a cutoff energy of 400 eV.
```

Comment:
Just to check, is this saying (or is it the case) that the reference energies we compare to are used PBE?